// Copyright 2025 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";
option optimize_for = LITE_RUNTIME;

package tabs_pb;
option java_package = "org.chromium.chrome.browser.tabs.proto";

message SplitCollectionState {
  optional fixed64 split_tab_id_high = 1;
  optional fixed64 split_tab_id_low = 2;
  optional int32 split_layout = 3;
  optional float split_ratio = 4;
}
